doctr/models/core.py

```diff
@@ -42,13 +42,22 @@ def __init__(
     def __call__(
         self,
         pages: List[np.ndarray],
+        rotate_document: bool = True,
         **kwargs: Any,
     ) -> Document:
 
         # Dimension check
         if any(page.ndim != 3 for page in pages):
             raise ValueError("incorrect input shape: all pages are expected to be multi-channel 2D images.")
 
+        # Detect document rotation and rotate pages
+        if rotate_document:
+            page_angles = []
+            for i,page in enumerate(pages):
+                page_angle = estimate_orientation(page)
+                page_angles.append(page_angle)
+                pages[i] = rotate_page(page,page_angle)
+
```

Comment:
in this PR, I suggest discarding the integration in the predictor and keeping this only about orientation estimation :+1:

Reply:
ok got it !!